core/db/migrate/20221123152807_add_shipping_category_to_spree_variants.rb

```diff
@@ -0,0 +1,5 @@
+class AddShippingCategoryToSpreeVariants < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :spree_variants, :shipping_category, index: true
```

Comment:
Should we make it a foreign key? We only have a few on the schema, but I recall we haven't added the rest only for BC concerns. Nice if we do the correct now and adapt the rest in the future.

Reply:
Yeah. Good point. I first had a foreign key but then saw that we haven't many in the schema. I will add one

Reply:
Done